webpack.config.js

```diff
@@ -0,0 +1,35 @@
+var path = require('path');
+var SRC_DIR = path.join(__dirname, '/client/src');
+var DIST_DIR = path.join(__dirname, '/client/dist');
+const MomentLocalesPlugin = require('moment-locales-webpack-plugin');
+
+module.exports = {
+  entry: `${SRC_DIR}/index.jsx`,
+  output: {
+    filename: 'bundle.js',
+    path: DIST_DIR
+  },
+  module: {
+    loaders: [
```

Comment:
Have you tested this webpack config to see if it actually bundles correctly? I'm using the latest version and it required some different values on the `module` property. Take a look at mine if this doesn't work as expected.

Reply:
Thanks for suggestion, you are right - my config didn't work. I updated the 'babel-preset-env' and "@babel/preset-react" versions n package.json, and webpack config file to follow your suggestions - and it fixed the issues.